ui/src/app/components/help-sidebar.tsx

```diff
@@ -524,15 +540,69 @@ export const HelpSidebar = fp.flow(withCurrentWorkspace(), withUserProfile(), wi
       return !icon.page || icon.page === helpContentKey || (criteria && icon.page === 'criteria');
     }
 
-    displayIcon(icon, i) {
-      const {criteria} = this.props;
+    displayFontAwesomeIcon(icon) {
+      const {criteria, runtimeStore} = this.props;
 
       return <React.Fragment>
-        {(criteria && icon.page === 'criteria') && <span data-test-id='criteria-count'
-                                                         style={styles.criteriaCount}>
-          {criteria.length}</span>}
-            <FontAwesomeIcon data-test-id={'help-sidebar-icon-' + icon.id} icon={icon.faIcon} style={icon.style} />
-          </React.Fragment> ;
+        {(criteria && icon.page === 'criteria') &&
+          <span data-test-id='criteria-count' style={styles.criteriaCount}>
+            {criteria.length}
+          </span>
+        }
+        <FontAwesomeIcon data-test-id={'help-sidebar-icon-' + icon.id} icon={icon.faIcon} style={icon.style} />
+      </React.Fragment> ;
+    }
+
+    displayRuntimeIcon(icon) {
+      const {runtimeStore} = this.props;
+
+      return <FlexRow style={{height: '100%', alignItems: 'center', justifyContent: 'space-around'}}>
+        <img data-test-id={'help-sidebar-icon-' + icon.id} src={proIcons[icon.id]} style={{...icon.style, position: 'absolute'}} />
+        {
+          runtimeStore.currentRuntime
+          && (
+              runtimeStore.currentRuntime.status === RuntimeStatus.Creating
```

Comment:
I think the verbosity can be greatly reduced here by just pulling out `status` as a local var ahead of all this

Reply:
`runtimeStore` may be null. As such, I can pull that out, but not `status`

Reply:
I wuld just make status null if currentRuntime is null

Reply:
can one do that with destructuring?

Reply:
I don't think you can, would just skip the destructuring.

```
const status = runtimeStore.currentRuntime && runtimeStore.currentRuntime.status;
```

or if you really like destructuring:

```
const {status} = runtimeStore.currentRuntime || {};
```